js/src/lib/interfaces/Libraries.ts

```diff
@@ -469,5 +470,11 @@ export const MODEL_LIBRARIES_UI_ELEMENTS: { [key in keyof typeof ModelLibrary]?:
 		repoUrl:  "https://github.com/huggingface/huggingface_sb3",
 		snippet:  stableBaselines3,
 	},
+	"ml-agents": {
+		btnLabel: "ml-agents",
+		repoName: "ml-agents",
+		repoUrl:  "https://github.com/huggingface/ml-agents",
+		snippet:  ml-agents,
```

Comment:
You are missing the snippet. Check out the others, they are functions with the code snippet to be shown

Reply:
Added thanks for the tip!